.github/workflows/build.yml

```diff
@@ -28,9 +29,94 @@ jobs:
         terraform init -backend=false
         terraform validate -no-color
       working-directory: terraform
-
+  
     - name: Deploy to dev environment
       uses: ./.github/workflows/actions
       with:
         environment_name: dev
         azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
+
+  deploy_test:
+    name: Deploy to test environment
+    needs: deploy_dev
+    runs-on: ubuntu-latest
+    environment: test
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 1.0.10
+
+    - name: Check formatting
```

Comment:
Formatting and validation were already done in dev, we probably don't need to do it again